app/code/Magento/SalesRule/Test/Mftf/Test/StorefrontCartPriceRuleDescriptionTest.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontCartPriceRuleDescriptionTest">
+        <annotations>
+            <features value="SalesRule"/>
+            <stories value="Create cart price rule"/>
+            <title value="Cart Price Rule description output"/>
+            <description value="Checking the output of the Cart Price Rule description"/>
+            <severity value="MAJOR"/>
+        </annotations>
+
+        <before>
+            <createData entity="ApiCartRule" stepKey="createSalesRule"/>
```

Comment:
Hi @p-ivanko. Good job. 
In order to persist test isolation (non-causing side effects by a test for other tests), we need to remove all entities created in the test. The common practice is to do it in the `<after>` section. 

Thanks.

Reply:
@rogyar, oh, sure, sorry.
Done